core/src/test/scala/unit/kafka/security/token/delegation/DelegationTokenManagerTest.scala

```diff
@@ -315,13 +314,20 @@ class DelegationTokenManagerTest extends QuorumTestHarness  {
       List()
     }
     else {
-      def authorizeToken(tokenId: String) = {
+      def authorizeToken(tokenId: String): Boolean = {
         val requestContext = AuthorizerUtils.sessionToRequestContext(hostSession)
         val action = new Action(AclOperation.DESCRIBE,
           new ResourcePattern(DELEGATION_TOKEN, tokenId, LITERAL), 1, true, true)
         aclAuthorizer.authorize(requestContext, List(action).asJava).asScala.head == AuthorizationResult.ALLOWED
       }
-      def eligible(token: TokenInformation) = DelegationTokenManager.filterToken(requestPrincipal, Option(requestedOwners), token, authorizeToken)
+      def authorizeRequester(owner: KafkaPrincipal): Boolean = {
+        val requestContext = AuthorizerUtils.sessionToRequestContext(hostSession)
+        val action = new Action(AclOperation.DESCRIBE,
```

Comment:
Do we have a test to exercise this method alone?  why are we using `DESCRIBE` Operation here?

Reply:
Good point, shoulde be `DESCRIBE_TOKENS`. I'll correct this. I believe the `DelegationTokenEndToEndAuthorizationWithOwnerTest` tests the relevant code path (that uses `DelegationTokenManager.filterToken`) but that uses the correct permissions (it sets them through the ACL command). Will get back to you with the test for this.

Reply:
This is still using `DESCRIBE` operation